.ci/docker/requirements-ci.txt

```
@@ -124,7 +124,7 @@ opt-einsum==3.3
 #Pinned versions: 3.3
 #test that import: test_linalg.py
 
-#pillow
+pillow==9.2.0
```

Can we limit it to less than 10? (So that newer Python runtime will get newer binary)
```suggestion
pillow>=9.2.0,<10.0.0
```